tools/psalm/psalm.xml

```diff
@@ -10,6 +11,13 @@
         </ignoreFiles>
     </projectFiles>
 
+    <extraFiles>
+        <!-- because of the structure of this project, it's not possible to
+            autoload phpbench attributes. Until that's fixed via a single `composer.lock`,
+            we can refer to the symbols provided by phpbench, manually -->
```

Comment:
TBH, I really really really dislike the `tools/` setup, but this is all the complaining you'll get for me while working on this specific patch :stuck_out_tongue: